apps/pollbook/backend/src/pollbook_package.ts

```diff
@@ -63,9 +63,32 @@ type PollbookPackageParseError =
     }
   | ReadFileError;
 
+export type ExternalToInternalPrecinctIdMapping =
+  | {
+      type: 'single-precinct';
+      precinctId: string;
+    }
+  | {
+      type: 'multi-precinct';
+      precinctIds: Record<string, string>;
+    };
+
+function trimZerosFromExternalPrecinctId(externalId: string): string {
+  const externalWardIdParseIntResult = safeParseInt(externalId);
+  if (externalWardIdParseIntResult.isErr()) {
+    throw new Error(`Non-numeric ward or district: ${externalId}`);
```

Comment:
Are ward or district IDs ever alphanumeric?

Reply:
in NH, they are always just numeric. this already doesn't apply to anywhere outside of NH, so I think it's OK to further tune it to NH